### DISCUSSION TITLE
Auto Saving Incoming snap

### DISCUSSION THREAD

**Discussion by GoatBan** [rodit/SnapMod#124] [Ideas & Feature Suggestions]
Is it possible to add an autosaving option like in snapt00ls? I found it annoying to press & save each time.

**Comment by RevealedSoulEven** [comment]
He already added an auto save message feature and then is option to configure the settings.

Just select "snaps" over there and here you go!

**Reply by GoatBan** [reply]
> He already added an auto save message feature and then is option to configure the settings.
> 
> Just select "snaps" over there and here you go!

The feature isnt  working for me, it save the snap in chat & send a "snap saved in chat " notification

**Reply by rodit** [reply]
Yes, that's not for saving the actual snaps. I might add this feature.

**Comment by rodit** [comment]
Just to clarify, do you mean to save the actual image/video to your device automatically?

**Reply by RevealedSoulEven** [reply]
Btw any plan for creating tutorials?😂 Just kidding but you are great modder.

**Reply by rodit** [reply]
I might consider writing some kind of blog about this project haha - that would be quite fun and interesting.

**Comment by GoatBan** [comment]
> Just to clarify, do you mean to save the actual image/video to your device automatically?

Yes, automatically saving  the Snap (red/purple) to my device

**Reply by RevealedSoulEven** [reply]
Maybe you need to save it from a specific user right? Not from all I guess

**Reply by GoatBan** [reply]
> Maybe you need to save it from a specific user right? Not from all I guess

Yeah it can be an option, but saving from everyone is good too, I can sort them later

**Comment by rodit** [comment]
Added in 1.7.0 (global auto download and per-chat auto download option)

**Reply by GoatBan** [reply]
It work flawlessy, thanks a lot !
